nevergrad/functions/gym/test_gym_multi.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+import os
+from . import gym_multi
+
+
+def test_gym_multi() -> None:
+    if os.name != "nt":
```

Comment:
with UnsupportedExperiment in the function, this condition must be removed

Reply:
Done.

Reply:
well, it does not look like it :p you should be able to safely remove this condition now